Automattic/wp-calypso
# DevDocs: Hide Inline Help

#### Changes proposed in this Pull Request

Hides Inline Help in DevDocs.

#### Testing instructions

Visit `/devdocs` and verify that Inline Help doesn't appear.

cc @lancewillett 

Fixes #26457
